stellarphot/photometry/tests/test_photometry.py

```diff
@@ -22,10 +22,25 @@
 )
 from stellarphot.settings import ApertureSettings
 
+# Constants for the tests
+
 GAINS = [1.0, 1.5, 2.0]
 # Make sure the tests are deterministic by using a random seed
```

Comment:
Really not 'using a random seed', rather you are forcing a 'fixed seed for random number generator'

Reply:
That is true...